auth/test/TestMT50.m

```diff
@@ -0,0 +1,29 @@
+classdef TestMT50 < matlab.unittest.TestCase
+
+    methods (Test)
+
+        function test_encode_decode(testcase)
+            HMAC1 = [uint8(3); uint8(5)];
```

Comment:
nitpick: shorter line: uint8([3;5]).

If you editing this file in sublime...it will be quick. See the following links:

https://www.sublimetext.com/docs/3/multiple_selection_with_the_keyboard.html
https://www.youtube.com/watch?v=czQipWJA8EU